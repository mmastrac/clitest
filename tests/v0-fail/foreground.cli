#!/usr/bin/env clitest --v0

background {
    $ printf "hi\n"
    ! hi
}

$ exit 1
