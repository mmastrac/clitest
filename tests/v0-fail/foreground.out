Running <root>/foreground.cli ...

exit 1
---
(no output)
---
❌ FAIL: exit status: 1
 → exit 1

(background) printf "hi\n"
---
hi
---
✅ OK

<root>/foreground.cli FAILED
Error: exit status: 1

